### PR TITLE
[Feature] add file statistics function

### PR DESCRIPTION
**Pull request by liubingxing**
Adding the file stats function to analyse the files for each volume. 
It will read the inode file from snapshot dir and count the files for several size range.
We can view the file distribution for each volume on grafana.
image

We also add a cmd to control the file stats function enable, the default value is false.

### CODE REVIEW

File: metanode/partition_file_stats.go [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+		// sleep a while if iter StatsInterval count
+		if count > StatsInterval {
+			count = 0
+			time.Sleep(StatsSleep)
+		}
+	}
+}
+
+func (mp *metaPartition) startFileStats() {
+	checkTicker := time.NewTicker(fileStatsCheckPeriod)
```
Comment (leonrayang) [inline metanode/partition_file_stats.go:172]: 10s interval may cause cpu cost too much

Comment (liubingxing) [reply metanode/partition_file_stats.go:172]: The **fileStatsCheckPeriod** here is mainly used to detect the status change of mp (such as leader to follower). The period for **fileStats** is **fileStatsHandlePeriod** (default value is 15min).

File: metanode/partition_file_stats.go [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+		labels := map[string]string{
+			"partid":    fmt.Sprintf("%d", mp.config.PartitionId),
+			"volName":   mp.config.VolName,
+			"sizeRange": toString(FileSizeRange(i)),
+		}
+		exporter.NewGauge("fileStats").SetWithLabels(float64(val), labels)
+	}
+}
+
+func (mp *metaPartition) fileStats(rootDir string) {
```
[the remark this answers is not in the document]
Comment (liubingxing) [reply metanode/partition_file_stats.go:87]: > recommend use mp snapshot store process to get all inodes info, the io cost seems not nessesary

That's good advice, and I'll optimize the code and test it again later.

**Comment by liubingxing** [comment]
@leonrayang This pr has been updated with the following changes. Please review again, thank you very much.
1. Collect inodes information during the processing of storing mp snapshot;
2. Adjust the check interval from 10s to 30s;

**Comment by leonrayang** [comment]
@M1eyu2018  gofmt not pass

**Comment by liubingxing** [comment]
@leonrayang The ci-tests process is a bit abnormal, and the following files are not subject to this pr changes.
image

**Review by leonrayang — APPROVED** [review]

**Pull request merged**